ui/eureka/eureka.js

```diff
@@ -2135,17 +2390,18 @@ class EurekaTracker {
   }
 
   AddFlag(x, y, beforeText, afterText) {
+    let simplify = this.SimplifyText(beforeText, afterText);
+    if (simplify) {
+      beforeText = simplify;
+      afterText = '';
+    }
+
     // https://github.com/ravahn/FFXIV_ACT_Plugin/issues/160
```

Comment:
I think this block of code here removing other characters can just be removed as this has been fixed in the plugin.

Reply:
Oh really? that's awesome!!